docs/architecture/adr-0001-app-setup-and-goals.md

```diff
@@ -0,0 +1,168 @@
+# Context
+
+We need to create an application without a dependency on Raster Foundry that has
+the following capabilities:
+
+- run models on imagery
+- kick off model runs when it learns about new imagery
+- run models without knowing anything about the content of those models
+- summarize past model results
+- allow users to respond to model run events
+
+Additionally, we'd like the application to meet several design goals that aren't
+core to the pursuit of those features:
+
+- we should deploy continuously
+- infrastructure should be kept as light as possible
+
+These requirements are located roughly a third of the way from "core competency
+team improvement" to vanity.
+
+The main question this ADR seeks to answer is how the API works and how
+communication between the external world and the API and the API and the model
+runner will work. This ADR will not attempt to answer questions of what a user interface
+should accomplish, since that is not germane to the short-term (three month-ish)
+uses we envision for this project.
+
+## Specific Components
+
+### Summary
+
+TODO
```

Comment:
I think a good thing to put in the summary would be an enumeration of the models for V1:
 - `AOI` - spatial and image (e.g. required bands, resolution) filters, the models to run on imagery if it passes, and the SNS topic to subscribe to for updates
 - `Model` - something that can run against imagery and produce geojson/stac-label output
 - `Prediction` - an instance of a model running against imagery